### ISSUE TITLE
List recent reviews 

### ISSUE THREAD

**Issue by scheib** [Reviewable/Reviewable#79]
I'd like to see the pull requests I've recently worked on, even when the pull requests are closed. e.g. the last 5 reviews I've participated in.

**Comment by pkaminski** [comment]
What's the use case, and how is it related to reviewing code?  (Reviewable doesn't aspire to being a complete UI replacement for all situations, so it's OK to go back to GitHub for some rare or off-topic actions.)

**Comment by scheib** [comment]
The use case is I reviewed some code. I remember something about the
review, but I'd like to double check what I (or others) wrote. Where was
the last place I was working on reviews? reviewable.io. What's the easiest
way back to that review? Well, I would guess reviewable.io, recent reviews.

Another: I did some review, but it's not done yet, I'm waiting for a
response. I go back to check in on the review the next day and it's not
there. Where did it go? Oh, it was closed by the author, but it disappeared
from the list of reviews. Would be nice if I saw it just below the active
reviews as closed.
